main.css

```diff
@@ -0,0 +1,71 @@
+h1 {
+    font-family: Georgia, "Times New Roman", Times, serif;
+    font-size: 2em;
+    text-align: center;
+    
```

Comment:
You align the title just fine, but forgot to center the images. (Part of the style requirements.)